src/games/strategy/triplea/Constants.java

```diff
@@ -144,6 +144,7 @@
   String UNIT_TYPE_CARRIER = "carrier";
   String UNIT_TYPE_FIGHTER = "fighter";
   String UNIT_TYPE_BOMBER = "bomber";
+  String UNIT_TYPE_TAC_BOMBER = "tactical_bomber";
```

Comment:
- The constants class is not a cohesive design (https://en.wikipedia.org/wiki/Cohesion_(computer_science)), part of the idea of minimal scope and information hiding. If this constant is not used outside of test code, why have the production code be aware of it or even depend on it? I'm suggesting you move the constant to the one class that uses it, and making it's scope private. Otherwise we're building on a bad pattern.

- Regardless, existing usage is perhaps flawed. tactical_bomber is not a unit type on all of the test maps, particularly not lhtr